dataflow/operators/tags_classifier_model/setting.py

```diff
@@ -0,0 +1,37 @@
+all_tags = [
+    'Covid-19',
+    'Covid-19 Feedback On Hmg Support',
+    'Covid-19 Business Disruption',
+    'Covid-19 Employment',
+    'Covid-19 Request For Hmg Support',
+    'Covid-19 Opportunity',
+    'Covid-19 Exports',
+    'Investment',
+    'Covid-19 Cash Flow',
+    'Eu Exit',
+    'Covid-19 Future Expectations',
+    'Covid-19 Offers Of Support',
+    'Covid-19 Supply Chain/Stock',
+    'Tariffs',
+    'Export',
+    'Regulation',
+    'Covid-19 Investment',
+    'Movement Of People',
+    'Skills',
+    'Covid-19 Resuming Operations',
+    'Access To Finance',
+    'Free Trade Agreements',
+    'Tax',
+]
+
+tags_general = [
+    i for i in all_tags if not i.lower().startswith('covid') or i.lower() == 'covid-19'
+]
+tags_covid = [
+    i for i in all_tags if i.lower().startswith('covid') and i.lower() != 'covid-19'
+]
```

Comment:
```suggestion
tags_covid = [
    i for i in all_tags if i.lower().startswith('covid')
]
tags_general = [
    i for i in all_tags if i not in tags_covid
]
```

Just a suggestion for a slightly cleaner setup here - it reads a little nicer this way, and removes some redundant checks.

These possibly might make more sense as sets rather than lists, but can be your call.

Reply:
i define it like that because the category 'covid-19'  need to be in the tags_general, and tags_covid only contains sub-categories that are under the  'covid-19' category.

Reply:
Ah, you're right, I misread the or condition. It could still be this, I think ... 🤔 But perhaps it's just fine as-is :)

```suggestion
tags_covid = [
    i for i in all_tags if i.lower().startswith('covid') and i.lower() != 'covid-19'
]
tags_general = [
    i for i in all_tags if i not in tags_covid
]
```

Reply:
cool thanks! will update!